packages/@local/eslint-config/index.js

```diff
@@ -253,6 +250,54 @@ module.exports = {
         "global-require": "off",
       },
     },
+    {
```

Comment:
This long list of overrides will be removed in #551. We need for now after removing root `tsconfig.json`.